test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Admin/DatabaseAdminTests.cs

```diff
@@ -85,5 +85,25 @@ private LiveConnectionHelper.TestConnectionResult GetLiveAutoCompleteTestObjects
             requestContext.VerifyAll();
         }
 
+        /// <summmary>
+        /// Get database info test
+        /// </summary>
+        // [Fact]
```

Comment:
Why is the test commented out? Will you be filling in the real logic in a future iteration?